### PR TITLE
Prac 06 code review request

### PR DESCRIPTION
**Pull request by phuongnam2709**
@KaungKhantKyaw1st

### CODE REVIEW

File: prac_06/car.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+class Car:
+    """Represent a Car object."""
+
+    def __init__(self, fuel=0, name=""):
+        """Initialise a Car instance.
+
+        fuel: float, one unit of fuel drives one kilometre
+        """
+        self.fuel = fuel
```
Comment (KaungKhantKyaw1st) [inline prac_06/car.py:12]: you could add this comment ,amount of fuel available in the car to specify

File: prac_06/car.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        self._odometer = 0
```
Comment (KaungKhantKyaw1st) [inline prac_06/car.py:13]: and in here it could be shown private attribute to track the distance driven

File: prac_06/car.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        self.name = name
```
Comment (KaungKhantKyaw1st) [inline prac_06/car.py:14]: this part u can specify the naming of the car

File: prac_06/guitar.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+        return f"{self.name} ({self.year}) : ${self.cost:,.2f}"
+
+    def get_age(self):
+        """Return the age of the guitar."""
+        current_year = 2024
+        return current_year - self.year
+
+    def is_vintage(self):
+        """Return True/False if the guitar is vintage (50 or more years old) or not."""
+        return self.get_age() >= 50
```
Comment (KaungKhantKyaw1st) [inline prac_06/guitar.py:20]: The class is well-structured and easy to understand.
 Adding error handling and dynamic date fetching will improve robustness.
 Enhancing string output could make the class more informative when printed.
 Consider writing unit tests to ensure correctness, especially for edge cases like future years.

**Review by KaungKhantKyaw1st — COMMENTED** [review]
overall the code is well-structured and easy to understand
